src/snyk/common/configuration/configuration.ts

```diff
@@ -154,6 +156,10 @@ export class Configuration implements IConfiguration {
     return this.defaultOssApiEndpoint;
   }
 
+  get snykLearnEndpoint(): string {
```

Comment:
It would be better to have this as a constant on LearnService instead, since it's the only place where it gets called from :)